Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -705,11 +886,18 @@ def parseInput(String[] cliArgs){
 
 	// cli overrides defaults set in 'packageBuildOutputs.properties'
 	props.generateWaziDeployAppManifest = (opts.wd) ? 'true' : props.generateWaziDeployAppManifest
-	props.addExtension = (opts.ae) ? 'true' : props.addExtension
+	props.addExtension = (opts.ae) ? 'true' : ((props.addExtension) ? (props.addExtension) : 'true')
 	props.publish = (opts.p) ? 'true' : props.publish
 	props.generateSBOM = (opts.sbom) ? 'true' : props.generateSBOM
 
+	if (opts.o) props.owner = opts.o
+
 	props.verbose = (opts.verb) ? 'true' : 'false'
+	
+	if (opts.af) {
+		props.applicationFolderPath = opts.af
+		if (opts.bp) props.baselinePackageFilePath = opts.bp
```

Comment:
I would rather like to stay with baselinePackage, because it is actually referencing the tar file.

```suggestion
		if (opts.bp) props.baselinePackage = opts.bp
```

Reply:
`baselinePackageFilePath` references the path to the file being the baseline package. I'd better keep it explicit like this.

Reply:
@M-DLB Could it be `baselinePackageFile` ? Because it is the file not the path that contains the file. I think that has confused me.